patches/api/0449-Add-required-plugins-config-option.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tamion <70228790+notTamion@users.noreply.github.com>
+Date: Sun, 26 Nov 2023 09:02:09 +0100
+Subject: [PATCH] Add required-plugins config option
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index 884902cb7f86c0b56594ccafc7d05c6c7a23ab53..2f1249ece0f5d6b5476f31e61a1c03155a3be2c3 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -985,6 +985,15 @@ public final class Bukkit {
+     public static @Nullable ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull org.bukkit.generator.structure.StructureType structureType, @NotNull org.bukkit.map.MapCursor.Type mapIcon, int radius, boolean findUnexplored) {
+         return server.createExplorerMap(world, location, structureType, mapIcon, radius, findUnexplored);
+     }
++
++    /**
++     * Checks if the Server is currently Reloading
++     *
++     * @return if the Server is Reloading
++     */
++    public static boolean isReloading() {
++        return server.isReloading();
++    }
+     // Paper end
+ 
+     /**
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index d0c634629aa0b6bac0da93655dd86ad3aea0ce30..90497df99691d3e42fceaab4685db9971f77e44d 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -839,6 +839,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+      *      org.bukkit.generator.structure.StructureType, int, boolean)
+      */
+     @Nullable ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull org.bukkit.generator.structure.StructureType structureType, @NotNull org.bukkit.map.MapCursor.Type mapIcon, int radius, boolean findUnexplored);
++
++    /**
++     * Checks if the Server is currently Reloading
++     *
++     * @return if the Server is Reloading
```

Comment:
```suggestion
+     * Checks if the server is currently reloading
+     *
+     * @return if the server is reloading
```
Might need a better explanation

Reply:
i mean what do you suggest? "Checks if the server is currently reloading." seems self explanatory